app/authentication/auth.py

```diff
@@ -50,6 +49,7 @@ def requires_auth():
         raise AuthError("Invalid token: iss not provided", 403)
 
     if client == current_app.config.get('ADMIN_CLIENT_USER_NAME'):
+        g.service_id = current_app.config.get('ADMIN_CLIENT_USER_NAME')
```

Comment:
im not sure we can meaningfully extract that info cos i think current_service is only stored on the admin